chapters/en/slides/module7/module7_04.Rmd

```diff
@@ -0,0 +1,342 @@
-nLl2tyxfmqwavOfp0qGcnZm2kbl-vvhvyBpHWQObPiHsVRI9x56OHv6x+XUf9j8gG7D+X8HzbYStc+Ja+-+AWG9liP-h-hfoe6yWXbtkAzg7QF3ZJuk0Mb6jamluBawUpBnnre+iPlMCHu-5i1wCIoAUpYQBLBsJATWWjhpbTEWlsQJHOXhKbRIBPXt47RSvjvwgYBiluAFmWjHm6ytebHsthu8KVE96kI--oRaDOyguj7GggCDR7047AKN7d295he4tqP7iV4sBvaGcS56aINH7IB9wGs5DaOViJznYiTuD6IgZTq9TMBO1NchmMCgaT56EQhiuqpAnJu+7FOoxOwAX+nft4bBe5SJUgVijSAY7QBfPkP7cBahpUBXmDBmqp+qV3uIHumVUjD61mg-uc6pwCWCrT46oPiJpU2aaOn7rUsznBjBAegdI6yurgTL7uBeHDwF5eCPtEwfeu5kLgd+EGB059GcRDiyw0qZhKahBRgU+4xMtAvoBeA1Gpa47OveEUGvQgllIbwKO2vuo-QZ1oz6ZB8DIX6-wcAKQCkwuSL-CNBUZpYEUYK1PgGa4JhombgWV0GmrW+AwQFSeBRNuJjMUlipAAsk09qMSm02gar7TEdorkzOA7Dvt5cBeHEoKoGGziIGfeQuCr5peoFjrpheGLp+LVB1Xl45u6CEkZBGygPBuoBeAQajgrUYtEQL0y7waLLw+VrL75RufXsOADe9DqjYnO4vjpTgh+IiQG9iE1jL4whT-h67LaDYDOAig8btZjduO1iObfekgW2qJUQdviwP+uIcP6tu4Rg0DLe3MsIzkuYqrd5dW50vq6L82-nz4Mh0Xl9zA0fwcJpFUTDvYHSu-UjpaiUQuIf6Ch1wV9y9KyjsKARUaaE754O0GNIaJasht4Dg2MjvKFhBAmq8zXAmgMTTYGtjjM5UhqsuiE1SumB8afmefgqHy8X3FlZkyBkK-hB+y1DxgahRXq15IqSaKmbp01vs6FmUhft4FeW5fqTqnS0gbjh58LgcT6hhxTN8GIWp+K-5NeH-rv5-uX1vSFGhY-p4qlQlHlHafeMJupbMQiQQ-jJB50nUQye6QUdQvceYZUBr+1Gm+4Iu7hl+5lhUhK166hgVnN71hSYSzSF+nnvF5VobYXeaIgbQUYYls6XiV5hONNKh7rGhodC4lQM4CQSigLALgApeTwVD5Zh1YakE7Kv-pHANhy4TG5rh9OJHLcyaaJaH7OSQZ9J62OngtDQhdptJbWY6IETDKAI2p2HzQ9gclSZekIUNJL+A4QMLx+zCCYGasSIFg7jhkPnSLAugMKcFes5Fn04ChigZUCoG3ujxLGgfgZOFumqOCyKQyhtN-4kOuYTUEl6BYSj77BMwY8EJq-Moyx3hVYjI6JhjYboAv+HAJY5eW14TMI4+80CR4LhJDoxEkR0lHUHLowQCS5+eaIEcEV+kFLaGeCdIJ751hnAUBGUCYHuhCKeyemYD8I7dhqgKWYAcWEsGzBLHZbq24trY0m+iqUIPh2If2EoRIAOarSezQUm62EwIfuhEeUIgxb7qZVg7rERbFiBHhBQoAiyYBGPoBa9213tuI-h1llp5ZuolGminOx4V5EIWSPm1RGQNnMiYbieAcFG6CI+pt5hR-mNJ58h0UYpF-8ykeXh2GKFiujnQdwCDJAh-2hhbWhFqInZ-KHXl7S94-EV8GgOJFtG5K2RsOQRUR6FhS4D2Lkbza8Robs1Hyehngo58iLDPkZaoF2HZF6RKIWK5XsjPLhK8hZck6HdKiJuoCvu9xu06xysEd5iYmNRrOohhY7sUC2MLXJO7RAVaOZ6+eCOvyEchCYq76O2ldjS48uq0ZcYXifPP9xj8qNvEEzCktkmb4iq9tWL8hOIQt59MH0bCAtcJIU9RsGf0eBaG4wdnlGgx5nEJotcTNPPBYan7jDE6U4Fk0QhByEZcb4mdJAZDogKmOdh9Azfs4at+dDgrJYxHBlX7-uWIYBG5eBIYpYgozWnMHOgrgJKHSRQsktIOhr0cz41KlziAhveuQtDFfhS4BL6DmmIQdQCxI0adowupNnC6T2hsBRGyY24Zrq0yvfv5j9oDoRZEKRYdoS6CEvgYyqohhDPBEuaMsfTSo0csXL4XUL4Ue6QquHrpGFsIkBLEIKUka7yz8vVpcH5RvKnFFTAzYa5ZeqrDBdjs+LXo-bJad7rIKHhb5n7ENio-k2FCRLYSHGmy2Br6E6Bkcd6Ya+mLsDGWRAkT8GxeXngl7Sgo-JbafeP0boIzhZYnOHLYGYOw5PhFHia7Gw5KFAAkmLsYWISRO4XMJ0+0cfcryRvcDFHyxUwJg7ncxhCqjKKMstMG9UXcXSKZ6xak-YRRnXoz7xxrMimEj4ygWgasEhvtgazxYqnhGEO50hODsOK8UxEf23ihFQax+8bP4BirzhWpLhLUa6qbx2VrEbuGlcUVZBB3ps9GyutsVdaF+boZ6qI2esJ3Zs6EcTGGu838aG6Nxj8UGSs+0VvkGux78fCq6B3pjt4HhTMQXFtR03B1Etxw3oRD3BekUDbisHvpt5lK+4dLSOh+MYLHDCY0bkameBRgkDfggiP5HJA2AYgnsuVYWLS4SfgAY6-xkbqybjuDDFO5VoWgIoAFI10ciGUhsbNrFtKMocvF8Jebk2YnQRHA14dxJTttH08JSsJzvCuTM4Fw+Cibr7v2ifhQ4MkohqAnuxsPqMo0hkCTfIPxAvuvHgeIGgTRAo9ODHriGEoWiH9SOcdu4GJsIX9Y32mgBdHixHMRvBSx50mwF6izXlQnDxVms2bbBaqBoEbqCaqi7eYWUctgxUkTnnEGxlxoe5B0-YMkyb+V8eWagWSWpl4NRJtFoG+JsftkEPSyYKiS-I5iSEkvk3MYbRoJ0tOJFVJjIV9y24RHEChi6TSeKrw0uroxZ9xKCojFvR+MMW4OGXoTNIZx4rN37FqMieeiM2ZcvrGDxq8QiQAAurIBAAA)
```

Comment:
Does this need to be so much text here? If so fine, just wanted to point out in case its a typo.

Reply:
Yes the vegaeditor encodes the entire spec as a string, so it is the included data that makes this link so big.